packages/web-components/src/components/ic-card/ic-card.spec.ts

```diff
@@ -183,4 +183,26 @@ describe("ic-card", () => {
 
     expect(page.root).toMatchSnapshot();
   });
+
+  it("should call 'setFocus' when card as a button is focused", async () => {
+    const page = await newSpecPage({
+      components: [Card],
+      html: `<ic-card heading="Card" message="This is a clickable card rendered as a button" clickable=true></ic-card>`,
+    });
+
+    await page.rootInstance.focus;
+
+    await page.rootInstance.setFocus().toHaveBeenCalled;
+  });
+
+  it("should call 'setFocus' when card as a button is focused", async () => {
```

Comment:
this seems very similar to the test above apart from href prop. Should the description be changed?

Reply:
Ah yeah, meant to change it to say link